.changeset/strange-socks-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@apollo/graphql-testing-library": patch
+---
+
+Adds `createHandlerFromSchema`
```

Comment:
I see you've also added a bunch of exports for utilities. Would that be worth calling out here or in a separate changeset?

I've also noticed that `createHandlerFromSchema` is more-or-less the same as `createHandler` was before this since the old `createHandler` took a `schema` as an argument. Would it be worth calling out that breaking change to specify that `createHandler` now accepts type defs and provides a bunch of the default resolvers for you?